### PR TITLE
Upgrade plugin to Sylius 1.13 compatibility

### PR DESCRIPTION
**Pull request by JanPalen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,9 +43,13 @@ This **open-source plugin was developed to help the Sylius community**. If you h
...
 ## Installation
 ```bash
-$ composer require bitbag/blacklist-plugin
+composer require bitbag/blacklist-plugin -W
 ```
-    
+
+You will see error during cache:clear operation. It will disappear after whole installation process.
+
+![Origin](doc/images/installation-error.png)
```
Comment (senghe) [inline README.md:51]: How about running the composer command with `--no-scripts` switch, and let user to run cache:clear at the end of process?

File: README.md [hunk 2]
```diff
@@ -200,34 +214,52 @@ Override Customer form template (`@SyliusAdminBundle\Customer\_form.html.twig`)
...
 
 **Note:** If you are running it on production, add the `-e prod` flag to this command.
 
+Update your database schema:
+
+```bash
+doctrine:schema:update --dump-sql
+```
+
+If the list only includes changes for updating the database by adding 'fraud_status,' you can use:
```
Comment (senghe) [inline README.md:229]:
```suggestion
If the list includes only changes for updating the database by adding 'fraud_status,' you can use:
```

File: README.md [hunk 3]
```diff
@@ -200,34 +214,52 @@ Override Customer form template (`@SyliusAdminBundle\Customer\_form.html.twig`)
...
+
+```bash
+doctrine:schema:update -f
+```
+
+If there is other changes make sure that will not destroy your database schema.
```
Comment (senghe) [inline README.md:235]:
```suggestion
If there are another changes, please make sure they will not destroy your database schema.
```

File: doc/functionalities.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+---
+A plugin designed to counteract suspicious behavior by blocking purchases using automated rules or manual customer behavior analysis.
+
+Customers are divided into three types:
+
+- blacklisted → blocked, cannot make purchases
+- whitelisted → trusted, not checked by the plugin during purchases
+- neutral → regular customer who can make purchases and is checked by the plugin with every order
+
+The customer's status can be changed manually in the customer edit panel or automatically using plugin functionalities.
```
Comment (senghe) [inline doc/functionalities.md:16]: In admin panel I guess?

File: tests/Application/.env [hunk 5]
```diff
@@ -21,14 +21,10 @@ JWT_PUBLIC_KEY=%kernel.project_dir%/config/jwt/public.pem
...
 
-###> symfony/swiftmailer-bundle ###
-# For Gmail as a transport, use: "gmail://username:password@localhost"
-# For a generic SMTP server, use: "smtp://localhost:25?encryption=&auth_mode="
-# Delivery is disabled by default via "null://localhost"
-MAILER_URL=smtp://localhost
-###< symfony/swiftmailer-bundle ###
+###> symfony/mailer ###
+MAILER_DSN=null://null
+###< symfony/mailer ###
```
Comment (senghe) [inline tests/Application/.env:26]: Why that changes?

**Review by senghe — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (JanPalen) [reply README.md:51]: Done. It's resolve issue

File: README.md (hunk 3 above)
Comment (JanPalen) [reply README.md:235]: done

File: README.md (hunk 2 above)
Comment (JanPalen) [reply README.md:229]: done

File: doc/functionalities.md (hunk 4 above)
Comment (JanPalen) [reply doc/functionalities.md:16]: Right. I added that information.

File: README.md [hunk 6]
```diff
@@ -43,9 +43,9 @@ This **open-source plugin was developed to help the Sylius community**. If you h
 
 ## Installation
 ```bash
-$ composer require bitbag/blacklist-plugin
+composer require bitbag/blacklist-plugin -W --no-scripts
```
Comment (senghe) [inline README.md:46]: But... why `-W` ?

File: README.md [hunk 7]
```diff
@@ -200,34 +210,52 @@ Override Customer form template (`@SyliusAdminBundle\Customer\_form.html.twig`)
...
 
 **Note:** If you are running it on production, add the `-e prod` flag to this command.
 
+Update your database schema:
+
+```bash
+doctrine:schema:update --dump-sql
+```
+
+If the list includes only changes for updating the database by adding 'fraud_status,' you can use:
```
Comment (senghe) [inline README.md:225]:
```suggestion
If the list includes only changes for updating the database by adding 'fraud_status' you can use:
```

**Review by senghe — APPROVED** [review]
Approved in advance. But please remove `-W` and the typo before switching task for testing.

File: README.md (hunk 6 above)
Comment (JanPalen) [reply README.md]: Yesterday I got problems without that flag. But today it is ok. Done

File: README.md (hunk 7 above)
Comment (JanPalen) [reply README.md]: Done

File: composer.json [hunk 8]
```diff
@@ -5,14 +5,15 @@
     "description": "Blacklist plugin for Sylius.",
     "license": "MIT",
     "require": {
-        "php": "^8.0",
-        "sylius/sylius": "~1.11.0 || ~1.12.0"
+        "php": "^8.0 || ^8.1 || ^8.2 || ^8.3",
+        "sylius/sylius": "~1.12.0 || ~1.13.0",
+        "doctrine/annotations": "^2.0.0"
```
Comment (christopherhero) [inline composer.json:10]: `"doctrine/annotations": "^2.0.0"`
It seems to me that this is unnecessary but I could be wrong

**Comment by senghe** [comment]
I've tested it both on Sylius 1.13 and 1.12 - the installation proceeds with no problem. I've also updated the README.md installation part to remove some misundersandings.

**Pull request merged**